### PR TITLE
feat(devtools): expose send function

### PR DESCRIPTION
**Pull request by yackinn**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/elf/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/devtools/src/lib/devtools.ts [hunk 1]
```diff
@@ -42,13 +42,21 @@ declare global {
...
+}
+
 export function devTools(options: DevtoolsOptions = {}) {
   if (!window.__REDUX_DEVTOOLS_EXTENSION__) return;
 
   let lock = false;
   const instance = window.__REDUX_DEVTOOLS_EXTENSION__.connect(options);
   const subscriptions = new Map<string, Subscription>();
 
+  externalEvents$.subscribe((action) => send(action));
```
Comment (NetanelBasal) [inline packages/devtools/src/lib/devtools.ts:58]: add to subscriptions

File: packages/devtools/src/lib/devtools.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-  getStoresSnapshot,
   getRegistry,
-  Store,
   getStore,
+  getStoresSnapshot,
+  registry$,
+  Store,
 } from '@ngneat/elf';
+import { Observable, Subject, Subscription } from 'rxjs';
+import { skip } from 'rxjs/operators';
```
Comment (NetanelBasal) [inline packages/devtools/src/lib/devtools.ts:10]: remove unused imports

File: packages/devtools/src/lib/devtools.ts (hunk 1 above)
Comment (NetanelBasal) [reply packages/devtools/src/lib/devtools.ts:58]: externalEvents$.subscribe(send);

File: packages/devtools/src/lib/devtools.ts (hunk 2 above)
Comment (yackinn) [reply packages/devtools/src/lib/devtools.ts:10]: These imports are being used.

**Pull request merged**